Scripts/Script0000_DDL.sql

```diff
@@ -52,9 +62,15 @@ CREATE TABLE public.ett_employee_metrics (
 	used_pto float8 NULL,
 	the_year int4 NOT NULL,
 	billable_target_to_date float8 NULL,
+	updated_at TIMESTAMPTZ NOT NULL DEFAULT NOW(),
```

Comment:
Technically we should be creating new scripts and adding things to the database that way instead of updating original scripts for versioning purposes